doc/design/show_train_stmt.md

```diff
@@ -0,0 +1,52 @@
+# Show Train Statement in SQLFlow
+
+## Background
+
+Like using other SQL engines, people may want to access the table metadata (say, the  table creating statement) using a SQL query. Usually they will type a statement like:
+```
+SHOW CREATE TABLE `test`;
+```
+which will produce:
+```
+mysql> SHOW CREATE TABLE `test`;
++-------+------------------------------------------------------------------+
+| Table | Create Table                                                                                                                                                                                                                                                          |
++-------+------------------------------------------------------------------+
+| test  | CREATE TABLE `test` (
+  `sepal_length` float DEFAULT NULL,
+  `sepal_width` float DEFAULT NULL,
+  `petal_length` float DEFAULT NULL,
+  `petal_width` float DEFAULT NULL,
+  `class` int DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci |
++-------+-----------------------------------------------------------------+
+```
+
+It worth for SQLFlow to do the same thing. On SQLFlow platform, user may create a lot of models with a minor set of parameters different with each other. After a while, the  user may hardly tell what the differences are if he hadn't record the information else where (like in model name).  We would like to provide a formal way to solve this problem. Here we introduce the SHOW TRAIN feature.
+
+## Grammar
+```
+SHOW TRAIN table_where_model_stored;
```

Comment:
`SHOW` and `TRAIN` are all verb, it does not comply with grammar rules.
`SHOW MODEL` or `DESCRIBE MODEL` seems better?

Reply:
We finally decided to use `SHOW TRAIN` for it more like SQL grammer`SHOW TRAIN TABLE` etc.

Reply:
There are two considerations. First, `SHOW CREATE` in SQL is to print the query that create this table, so, like that, we want `SHOW TRAIN` to print the model training query. Secondly, `DESCRIBE model` is a valid SQL query which print the colum meta. So if we use this form, it will be a prefix of our extended syntax, this may make our parsing process more complicated.